pkg/addons/adminconsole/static/values.tpl.yaml

```diff
@@ -19,3 +19,6 @@ passwordSecretRef:
     name: kotsadm-password
 service:
     enabled: false
+extraEnv:
```

Comment:
is this needed anymore if we're adding it on generate helm values? doesn't look like we include the others so we should be consistent and maybe not include it here